client/css/StyleSelector.scss

```diff
@@ -0,0 +1,13 @@
+.stylesAvailable {
+  margin: 10px 0;
+  display: inline-flex;
+  flex-wrap: wrap;
+  height: 300px;
+  align-content: flex-start;
+}
+
+.styleThumbnail {
+  margin: 5px 1%;
+  height: 30%;
```

Comment:
😢  percentages. 30% of what?

Reply:
Not sure why my height is a percentage, I think I was just testing different css options and forgot to change this when I commited. My new commit will fix this.

But I do use percentages for the width.
This is actually for my implementation of having only 4 thumbnails per row. I set the thumbnail container to be a flexbox row with flexwrap on, then I set the width of each thumbnail to be the total width of the container divided by 4.